internal/summoner/acquire/testdata/jsonutils_test.TestContextArrayFix.It_rewrites_the_jsonld_context_if_it_is_not_an_object.received.json

```diff
@@ -0,0 +1 @@
+"{\n        \"@context\": {\"@vocab\":\"https://schema.org/\",\"adms\":\"https://www.w3.org/ns/adms#\",\"dcat\":\"https://www.w3.org/ns/dcat#\",\"dct\":\"https://purl.org/dc/terms/\",\"foaf\":\"https://xmlns.com/foaf/0.1/\",\"gsp\":\"https://www.opengis.net/ont/geosparql#\",\"locn\":\"https://www.w3.org/ns/locn#\",\"owl\":\"https://www.w3.org/2002/07/owl#\",\"rdf\":\"https://www.w3.org/1999/02/22-rdf-syntax-ns#\",\"rdfs\":\"https://www.w3.org/2000/01/rdf-schema#\",\"schema\":\"https://schema.org/\",\"skos\":\"https://www.w3.org/2004/02/skos/core#\",\"spdx\":\"https://spdx.org/rdf/terms#\",\"time\":\"https://www.w3.org/2006/time\",\"vcard\":\"https://www.w3.org/2006/vcard/ns#\",\"xsd\":\"https://www.w3.org/2001/XMLSchema#\"},\n     \"@type\":\"bar\",\n      \"SO:name\":\"Some type in a graph\"\n    }"
```

Comment:
wait a second, should I be checking these things in? How is it supposed to work?